### PR TITLE
expose keys-dev as a preference

### PR DESCRIPTION
**Pull request by spencerstock**
### _Summary_

Expose keys dev: keys-dev.coinbase.com

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>1</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>2</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: packages/wallet-sdk/src/util/validatePreferences.ts
Comment (spencerstock) [inline packages/wallet-sdk/src/util/validatePreferences.ts:20]: Should we validate this?

File: packages/wallet-sdk/src/util/validatePreferences.ts
Comment (cb-jake) [inline packages/wallet-sdk/src/util/validatePreferences.ts:14]: wallet.coinbase.com uses `localhost:3005/connect` for local development.

File: packages/wallet-sdk/src/util/validatePreferences.ts
Comment (cb-jake) [reply packages/wallet-sdk/src/util/validatePreferences.ts:20]: Might not be a bad idea. The config has the field listed as deprecated and for internal use only.

**Review by cb-jake — COMMENTED** [review]
Approved but if we add to the validate preference util we should probably include localhost as well.

**Comment by spencerstock** [comment]
> Approved but if we add to the validate preference util we should probably include localhost as well.

After thinking on it more I prefer to remove validation for flexibility - it's pretty apparent what this does and the docs are specific enough I think

**Review by cb-jake — APPROVED** [review]

**Pull request merged**
